src/pybind/mgr/tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
     py3,
-    py37,
```

Comment:
i don't think we need to backport this commit.

Reply:
should I drop it from the quincy backport as well?

Reply:
> should I drop it from the quincy backport as well?

no, we should not